qiita_pet/templates/private_studies.html

```diff
@@ -3,13 +3,18 @@
 <link rel="stylesheet" href="/static/vendor/css/jquery.dataTables.css" type="text/css">
 
 <script src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<script src="/static/vendor/js/jquery.dataTables.plugin.natural.js"></script>
 
 <script type="text/javascript">
 var current_study;
 
 $(document).ready(function() {
-    $('#user-studies-table').dataTable({"order": [[ 1, "asc" ]]});
-    $('#shared-studies-table').dataTable({"order": [[ 1, "asc" ]]});
+        $('#user-studies-table').dataTable({
+            order: [[ 1, "asc" ]],
+            columnDefs: 'natural'});
+        $('#shared-studies-table').dataTable({
+            order: [[ 1, "asc" ]],
+            columnDefs: 'natural'});
```

Comment:
Can you drop your environment (`qiita_env drop`) and create a new test environment (`qiita_env make --no-load-ontologies` with TEST_ENVIRONMENT=TRUE in your qiita_config)? If you do that, then create two studies on the test@foo.bar user, then change `number_samples_collected` on one of the studies using psql to, e.g., 210, you should be able to replicate the test I'm running here